master/package.py

```diff
@@ -19,42 +19,13 @@
     steps.Git(
         name="Julia checkout",
         repourl=util.Property('repository', default='git://github.com/JuliaLang/julia.git'),
-        mode='incremental',
-        method='clean',
+        mode='full',
+        method='fresh',
```

Comment:
Will this make submitting nuke jobs unnecessary?

Reply:
Yes, hopefully.